tutor/specs/e2e/student-task.e2e.ts

```diff
@@ -0,0 +1,93 @@
+import faker from 'faker'
+import { visitPage, setRole, setTimeouts } from './helpers'
+
+describe('Student Tasks', () => {
+
+    beforeEach(async () => {
+        await page.setViewportSize({ width: 1200, height: 720 })
```

Comment:
1200x720 seems like a strange size.  Is there a reason for it?    Also, how about using the sizes you just defined above as SCREENS instead of hardcoding it

Reply:
mmm. good catch. I cherry picked from the previous PR and I think I forgot to change this hardcode values.